tasks/configuration.yml

```diff
@@ -1 +1,134 @@
 ---
+
+# - name:
+- name: configure php timezone
+  lineinfile:
+    path: /etc/opt/rh/rh-php71/php.ini
+    regexp: '^date.timezone( )?='
+    line: 'date.timezone = Europe/Zurich'
+  notify: restart php-fpm
+
+- name: enable icinga2 ido feature
+  template:
+    src: templates/etc/icinga2/features-enabled/ido-mysql.conf.j2
+    dest: /etc/icinga2/features-enabled/ido-mysql.conf
+    owner: icinga
+    group: icinga
+    mode: 0640
+
+- name: Set httpd_can_network_connect_db flag on and keep it persistent across reboots
+  seboolean:
+    name: httpd_can_network_connect_db
+    state: yes
+    persistent: yes
+
+- name: check if icingaweb2 table in mysql exists
+  shell: 'mysql {{icinga2_web_database_name }} -e "SHOW TABLES;" | grep "icingaweb_user"'
+  register: icinga_master_imported
```

Comment:
Please stick to our naming convention defined in the standard for registered variables: https://docs.adfinis-sygroup.ch/public/ansible-guide/styling_guide.html#